diff --git a/qdarkstyle/style.qrc b/qdarkstyle/style_2.1.qrc
similarity index 95%
rename from qdarkstyle/style.qrc
rename to qdarkstyle/style_2.1.qrc
index ac14bc5..fc75405 100644
--- a/qdarkstyle/style.qrc
+++ b/qdarkstyle/style_2.1.qrc
@@ -1,5 +1,5 @@
 <RCC>
-  <qresource prefix="qss_icons">
+  <qresource prefix="qss_icons_2_1">
     <file>rc/up_arrow_disabled.png</file>
     <file>rc/Hmovetoolbar.png</file>
     <file>rc/stylesheet-branch-end.png</file>
@@ -40,7 +40,7 @@
     <file>rc/radio_unchecked_focus.png</file>
     <file>rc/radio_unchecked.png</file>
   </qresource>
-  <qresource prefix="qdarkstyle">
+  <qresource prefix="qdarkstyle_2_1">
       <file>style.qss</file>
   </qresource>
 </RCC>
diff --git a/qdarkstyle/style.qss b/qdarkstyle/style_2.1.qss
similarity index 85%
rename from qdarkstyle/style.qss
rename to qdarkstyle/style_2.1.qss
index 8f61972..1c2dd95 100644
--- a/qdarkstyle/style.qss
+++ b/qdarkstyle/style_2.1.qss
@@ -91,7 +91,7 @@ QCheckBox::indicator:unchecked:hover,
 QGroupBox::indicator:unchecked,
 QGroupBox::indicator:unchecked:hover
 {
-    image: url(:/qss_icons/rc/checkbox_unchecked.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_unchecked.png);
 }
 
 QCheckBox::indicator:unchecked:focus,
@@ -100,7 +100,7 @@ QGroupBox::indicator:unchecked:focus,
 QGroupBox::indicator:unchecked:pressed
 {
   border: none;
-    image: url(:/qss_icons/rc/checkbox_unchecked_focus.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_unchecked_focus.png);
 }
 
 QCheckBox::indicator:checked,
@@ -108,7 +108,7 @@ QCheckBox::indicator:checked:hover,
 QGroupBox::indicator:checked,
 QGroupBox::indicator:checked:hover
 {
-    image: url(:/qss_icons/rc/checkbox_checked.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_checked.png);
 }
 
 QCheckBox::indicator:checked:focus,
@@ -117,7 +117,7 @@ QGroupBox::indicator:checked:focus,
 QGroupBox::indicator:checked:pressed
 {
   border: none;
-    image: url(:/qss_icons/rc/checkbox_checked_focus.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_checked_focus.png);
 }
 
 QCheckBox::indicator:indeterminate,
@@ -127,25 +127,25 @@ QGroupBox::indicator:indeterminate,
 QGroupBox::indicator:indeterminate:hover,
 QGroupBox::indicator:indeterminate:pressed
 {
-    image: url(:/qss_icons/rc/checkbox_indeterminate.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_indeterminate.png);
 }
 
 QCheckBox::indicator:indeterminate:focus,
 QGroupBox::indicator:indeterminate:focus
 {
-    image: url(:/qss_icons/rc/checkbox_indeterminate_focus.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_indeterminate_focus.png);
 }
 
 QCheckBox::indicator:checked:disabled,
 QGroupBox::indicator:checked:disabled
 {
-    image: url(:/qss_icons/rc/checkbox_checked_disabled.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_checked_disabled.png);
 }
 
 QCheckBox::indicator:unchecked:disabled,
 QGroupBox::indicator:unchecked:disabled
 {
-    image: url(:/qss_icons/rc/checkbox_unchecked_disabled.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_unchecked_disabled.png);
 }
 
 QRadioButton
@@ -169,7 +169,7 @@ QRadioButton::indicator
 QRadioButton::indicator:unchecked,
 QRadioButton::indicator:unchecked:hover
 {
-    image: url(:/qss_icons/rc/radio_unchecked.png);
+    image: url(:/qss_icons_2_1/rc/radio_unchecked.png);
 }
 
 QRadioButton::indicator:unchecked:focus,
@@ -177,7 +177,7 @@ QRadioButton::indicator:unchecked:pressed
 {
   border: none;
   outline: none;
-    image: url(:/qss_icons/rc/radio_unchecked_focus.png);
+    image: url(:/qss_icons_2_1/rc/radio_unchecked_focus.png);
 }
 
 QRadioButton::indicator:checked,
@@ -185,7 +185,7 @@ QRadioButton::indicator:checked:hover
 {
   border: none;
   outline: none;
-    image: url(:/qss_icons/rc/radio_checked.png);
+    image: url(:/qss_icons_2_1/rc/radio_checked.png);
 }
 
 QRadioButton::indicator:checked:focus,
@@ -193,25 +193,25 @@ QRadioButton::indicato::menu-arrowr:checked:pressed
 {
   border: none;
   outline: none;
-    image: url(:/qss_icons/rc/radio_checked_focus.png);
+    image: url(:/qss_icons_2_1/rc/radio_checked_focus.png);
 }
 
 QRadioButton::indicator:indeterminate,
 QRadioButton::indicator:indeterminate:hover,
 QRadioButton::indicator:indeterminate:pressed
 {
-        image: url(:/qss_icons/rc/radio_indeterminate.png);
+        image: url(:/qss_icons_2_1/rc/radio_indeterminate.png);
 }
 
 QRadioButton::indicator:checked:disabled
 {
   outline: none;
-  image: url(:/qss_icons/rc/radio_checked_disabled.png);
+  image: url(:/qss_icons_2_1/rc/radio_checked_disabled.png);
 }
 
 QRadioButton::indicator:unchecked:disabled
 {
-    image: url(:/qss_icons/rc/radio_unchecked_disabled.png);
+    image: url(:/qss_icons_2_1/rc/radio_unchecked_disabled.png);
 }
 
 
@@ -280,41 +280,41 @@ QMenu::indicator {
 /* non-exclusive indicator = check box style indicator
    (see QActionGroup::setExclusive) */
 QMenu::indicator:non-exclusive:unchecked {
-    image: url(:/qss_icons/rc/checkbox_unchecked.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_unchecked.png);
 }
 
 QMenu::indicator:non-exclusive:unchecked:selected {
-    image: url(:/qss_icons/rc/checkbox_unchecked_disabled.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_unchecked_disabled.png);
 }
 
 QMenu::indicator:non-exclusive:checked {
-    image: url(:/qss_icons/rc/checkbox_checked.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_checked.png);
 }
 
 QMenu::indicator:non-exclusive:checked:selected {
-    image: url(:/qss_icons/rc/checkbox_checked_disabled.png);
+    image: url(:/qss_icons_2_1/rc/checkbox_checked_disabled.png);
 }
 
 /* exclusive indicator = radio button style indicator (see QActionGroup::setExclusive) */
 QMenu::indicator:exclusive:unchecked {
-    image: url(:/qss_icons/rc/radio_unchecked.png);
+    image: url(:/qss_icons_2_1/rc/radio_unchecked.png);
 }
 
 QMenu::indicator:exclusive:unchecked:selected {
-    image: url(:/qss_icons/rc/radio_unchecked_disabled.png);
+    image: url(:/qss_icons_2_1/rc/radio_unchecked_disabled.png);
 }
 
 QMenu::indicator:exclusive:checked {
-    image: url(:/qss_icons/rc/radio_checked.png);
+    image: url(:/qss_icons_2_1/rc/radio_checked.png);
 }
 
 QMenu::indicator:exclusive:checked:selected {
-    image: url(:/qss_icons/rc/radio_checked_disabled.png);
+    image: url(:/qss_icons_2_1/rc/radio_checked_disabled.png);
 }
 
 QMenu::right-arrow {
     margin: 5px;
-    image: url(:/qss_icons/rc/right_arrow.png)
+    image: url(:/qss_icons_2_1/rc/right_arrow.png)
 }
 
 
@@ -393,7 +393,7 @@ QScrollBar::handle:horizontal
 QScrollBar::add-line:horizontal
 {
     margin: 0px 3px 0px 3px;
-    border-image: url(:/qss_icons/rc/right_arrow_disabled.png);
+    border-image: url(:/qss_icons_2_1/rc/right_arrow_disabled.png);
     width: 10px;
     height: 10px;
     subcontrol-position: right;
@@ -403,7 +403,7 @@ QScrollBar::add-line:horizontal
 QScrollBar::sub-line:horizontal
 {
     margin: 0px 3px 0px 3px;
-    border-image: url(:/qss_icons/rc/left_arrow_disabled.png);
+    border-image: url(:/qss_icons_2_1/rc/left_arrow_disabled.png);
     height: 10px;
     width: 10px;
     subcontrol-position: left;
@@ -412,7 +412,7 @@ QScrollBar::sub-line:horizontal
 
 QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:on
 {
-    border-image: url(:/qss_icons/rc/right_arrow.png);
+    border-image: url(:/qss_icons_2_1/rc/right_arrow.png);
     height: 10px;
     width: 10px;
     subcontrol-position: right;
@@ -422,7 +422,7 @@ QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:on
 
 QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on
 {
-    border-image: url(:/qss_icons/rc/left_arrow.png);
+    border-image: url(:/qss_icons_2_1/rc/left_arrow.png);
     height: 10px;
     width: 10px;
     subcontrol-position: left;
@@ -459,7 +459,7 @@ QScrollBar::handle:vertical
 QScrollBar::sub-line:vertical
 {
     margin: 3px 0px 3px 0px;
-    border-image: url(:/qss_icons/rc/up_arrow_disabled.png);
+    border-image: url(:/qss_icons_2_1/rc/up_arrow_disabled.png);
     height: 10px;
     width: 10px;
     subcontrol-position: top;
@@ -469,7 +469,7 @@ QScrollBar::sub-line:vertical
 QScrollBar::add-line:vertical
 {
     margin: 3px 0px 3px 0px;
-    border-image: url(:/qss_icons/rc/down_arrow_disabled.png);
+    border-image: url(:/qss_icons_2_1/rc/down_arrow_disabled.png);
     height: 10px;
     width: 10px;
     subcontrol-position: bottom;
@@ -479,7 +479,7 @@ QScrollBar::add-line:vertical
 QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on
 {
 
-    border-image: url(:/qss_icons/rc/up_arrow.png);
+    border-image: url(:/qss_icons_2_1/rc/up_arrow.png);
     height: 10px;
     width: 10px;
     subcontrol-position: top;
@@ -489,7 +489,7 @@ QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on
 
 QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on
 {
-    border-image: url(:/qss_icons/rc/down_arrow.png);
+    border-image: url(:/qss_icons_2_1/rc/down_arrow.png);
     height: 10px;
     width: 10px;
     subcontrol-position: bottom;
@@ -531,7 +531,7 @@ QHeaderView::section
 }
 
 QSizeGrip {
-    image: url(:/qss_icons/rc/sizegrip.png);
+    image: url(:/qss_icons_2_1/rc/sizegrip.png);
     width: 12px;
     height: 12px;
 }
@@ -592,16 +592,16 @@ QToolBar {
 }
 
 QToolBar::handle:horizontal {
-    image: url(:/qss_icons/rc/Hmovetoolbar.png);
+    image: url(:/qss_icons_2_1/rc/Hmovetoolbar.png);
 }
 QToolBar::handle:vertical {
-    image: url(:/qss_icons/rc/Vmovetoolbar.png);
+    image: url(:/qss_icons_2_1/rc/Vmovetoolbar.png);
 }
 QToolBar::separator:horizontal {
-    image: url(:/qss_icons/rc/Hsepartoolbar.png);
+    image: url(:/qss_icons_2_1/rc/Hsepartoolbar.png);
 }
 QToolBar::separator:vertical {
-    image: url(:/qss_icons/rc/Vsepartoolbars.png);
+    image: url(:/qss_icons_2_1/rc/Vsepartoolbars.png);
 }
 
 QPushButton
@@ -691,13 +691,13 @@ QComboBox::drop-down
 
 QComboBox::down-arrow
 {
-    image: url(:/qss_icons/rc/down_arrow_disabled.png);
+    image: url(:/qss_icons_2_1/rc/down_arrow_disabled.png);
 }
 
 QComboBox::down-arrow:on, QComboBox::down-arrow:hover,
 QComboBox::down-arrow:focus
 {
-    image: url(:/qss_icons/rc/down_arrow.png);
+    image: url(:/qss_icons_2_1/rc/down_arrow.png);
 }
 
 QPushButton:pressed
@@ -730,25 +730,25 @@ QAbstractSpinBox:down-button
 }
 
 QAbstractSpinBox::up-arrow,QAbstractSpinBox::up-arrow:disabled,QAbstractSpinBox::up-arrow:off {
-    image: url(:/qss_icons/rc/up_arrow_disabled.png);
+    image: url(:/qss_icons_2_1/rc/up_arrow_disabled.png);
     width: 10px;
     height: 10px;
 }
 QAbstractSpinBox::up-arrow:hover
 {
-    image: url(:/qss_icons/rc/up_arrow.png);
+    image: url(:/qss_icons_2_1/rc/up_arrow.png);
 }
 
 
 QAbstractSpinBox::down-arrow,QAbstractSpinBox::down-arrow:disabled,QAbstractSpinBox::down-arrow:off
 {
-    image: url(:/qss_icons/rc/down_arrow_disabled.png);
+    image: url(:/qss_icons_2_1/rc/down_arrow_disabled.png);
     width: 10px;
     height: 10px;
 }
 QAbstractSpinBox::down-arrow:hover
 {
-    image: url(:/qss_icons/rc/down_arrow.png);
+    image: url(:/qss_icons_2_1/rc/down_arrow.png);
 }
 
 
@@ -779,18 +779,18 @@ QTabBar:focus
 }
 
 QTabBar::close-button  {
-    image: url(:/qss_icons/rc/close.png);
+    image: url(:/qss_icons_2_1/rc/close.png);
     background: transparent;
 }
 
 QTabBar::close-button:hover
 {
-    image: url(:/qss_icons/rc/close-hover.png);
+    image: url(:/qss_icons_2_1/rc/close-hover.png);
     background: transparent;
 }
 
 QTabBar::close-button:pressed {
-    image: url(:/qss_icons/rc/close-pressed.png);
+    image: url(:/qss_icons_2_1/rc/close-pressed.png);
     background: transparent;
 }
 
@@ -896,26 +896,26 @@ QTabBar::tab:right:!selected:hover {
 }
 
 QTabBar QToolButton::right-arrow:enabled {
-     image: url(:/qss_icons/rc/right_arrow.png);
+     image: url(:/qss_icons_2_1/rc/right_arrow.png);
  }
 
  QTabBar QToolButton::left-arrow:enabled {
-     image: url(:/qss_icons/rc/left_arrow.png);
+     image: url(:/qss_icons_2_1/rc/left_arrow.png);
  }
 
 QTabBar QToolButton::right-arrow:disabled {
-     image: url(:/qss_icons/rc/right_arrow_disabled.png);
+     image: url(:/qss_icons_2_1/rc/right_arrow_disabled.png);
  }
 
  QTabBar QToolButton::left-arrow:disabled {
-     image: url(:/qss_icons/rc/left_arrow_disabled.png);
+     image: url(:/qss_icons_2_1/rc/left_arrow_disabled.png);
  }
 
 
 QDockWidget {
     border: 1px solid #403F3F;
-    titlebar-close-icon: url(:/qss_icons/rc/close.png);
-    titlebar-normal-icon: url(:/qss_icons/rc/undock.png);
+    titlebar-close-icon: url(:/qss_icons_2_1/rc/close.png);
+    titlebar-normal-icon: url(:/qss_icons_2_1/rc/undock.png);
 }
 
 QDockWidget::close-button, QDockWidget::float-button {
@@ -941,39 +941,39 @@ QTreeView, QListView
 
 QTreeView:branch:selected, QTreeView:branch:hover
 {
-    background: url(:/qss_icons/rc/transparent.png);
+    background: url(:/qss_icons_2_1/rc/transparent.png);
 }
 
 QTreeView::branch:has-siblings:!adjoins-item {
-    border-image: url(:/qss_icons/rc/transparent.png);
+    border-image: url(:/qss_icons_2_1/rc/transparent.png);
 }
 
 QTreeView::branch:has-siblings:adjoins-item {
-    border-image: url(:/qss_icons/rc/transparent.png);
+    border-image: url(:/qss_icons_2_1/rc/transparent.png);
 }
 
 QTreeView::branch:!has-children:!has-siblings:adjoins-item {
-    border-image: url(:/qss_icons/rc/transparent.png);
+    border-image: url(:/qss_icons_2_1/rc/transparent.png);
 }
 
 QTreeView::branch:has-children:!has-siblings:closed,
 QTreeView::branch:closed:has-children:has-siblings {
-    image: url(:/qss_icons/rc/branch_closed.png);
+    image: url(:/qss_icons_2_1/rc/branch_closed.png);
 }
 
 QTreeView::branch:open:has-children:!has-siblings,
 QTreeView::branch:open:has-children:has-siblings  {
-    image: url(:/qss_icons/rc/branch_open.png);
+    image: url(:/qss_icons_2_1/rc/branch_open.png);
 }
 
 QTreeView::branch:has-children:!has-siblings:closed:hover,
 QTreeView::branch:closed:has-children:has-siblings:hover {
-    image: url(:/qss_icons/rc/branch_closed-on.png);
+    image: url(:/qss_icons_2_1/rc/branch_closed-on.png);
     }
 
 QTreeView::branch:open:has-children:!has-siblings:hover,
 QTreeView::branch:open:has-children:has-siblings:hover  {
-    image: url(:/qss_icons/rc/branch_open-on.png);
+    image: url(:/qss_icons_2_1/rc/branch_open-on.png);
     }
 
 QListView::item:!selected:hover, QListView::item:!selected:hover, QTreeView::item:!selected:hover  {
@@ -1056,7 +1056,7 @@ QToolButton:checked, QToolButton:pressed,
 
 /* the subcontrol below is used only in the InstantPopup or DelayedPopup mode */
 QToolButton::menu-indicator {
-    image: url(:/qss_icons/rc/down_arrow.png);
+    image: url(:/qss_icons_2_1/rc/down_arrow.png);
     top: -7px; left: -2px; /* shift it a bit */
 }
 
@@ -1071,7 +1071,7 @@ QToolButton::menu-button {
 }
 
 QToolButton::menu-arrow {
-    image: url(:/qss_icons/rc/down_arrow.png);
+    image: url(:/qss_icons_2_1/rc/down_arrow.png);
 }
 
 QToolButton::menu-arrow:open {
@@ -1155,11 +1155,11 @@ QHeaderView::section:checked
 
  /* style the sort indicator */
 QHeaderView::down-arrow {
-    image: url(:/qss_icons/rc/down_arrow.png);
+    image: url(:/qss_icons_2_1/rc/down_arrow.png);
 }
 
 QHeaderView::up-arrow {
-    image: url(:/qss_icons/rc/up_arrow.png);
+    image: url(:/qss_icons_2_1/rc/up_arrow.png);
 }
 
 
